### ISSUE TITLE
Improvements to make the puzzle more fun.

### ISSUE THREAD

**Issue by ChadeBibbins** [yanamal/html-hax#33]
I feel we could add more color to the puzzles. Give it a background color or colorful text. We could even add a score every time we get an answer right.

**Comment by yanamal** [comment]
I agree; I added some styling in https://github.com/yanamal/html-hax/commit/d626d10fb40474e99f9bfdb9b1c50fae3294276a, but I think it needs more, so I'll keep this issue open for now.
